02week/pigLatin.js

```diff
@@ -0,0 +1,77 @@
+"use strict";
+
+const assert = require("assert");
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+// FIND THE FIRST VOWEL ++++++++
+
+function findFirstVowel(word) {
+  const vowel = ["a", "e", "i", "o", "u"];
+  for (let j = 0; j < word.length; j++) {
+    if (vowel.indexOf(word[j]) !== -1) {
+      return j;
+    }
+  }
+}
+
+// PIGLATIN FUNCTION +++++++++++
+console.log("---- PIG LATIN ----");
+
+function pigLatin(word) {
+  word = word.toLowerCase().trim();
+  let firstVowel = findFirstVowel(word);
```

Comment:
Nice!
but this will treat words that do not have a vowel as if they started with a vowel.